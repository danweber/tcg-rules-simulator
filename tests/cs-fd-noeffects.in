VERSION:2

# the inherit of a facedown card doesn't exist. CS1-Paper's inherited gives 1 memory at turn start.  CS2-01 gives +1000 DP while you have a red.


FIELD1:Inherited,DW1-Van,FACEDOWN,CS1-Paper,CS1-Paper,FACEDOWN,CS2-01 Red,CS2-ShieldSmasher
GAME:P1 T3 HATCHING 5
ECHO: should have 5000 DP and 9000 DP
DUMP:DP

CMD:MAIN
STEP:AUTOANSWER:CAPTURE
ECHO: -- main, gain just 1 mem
DUMP:DP

