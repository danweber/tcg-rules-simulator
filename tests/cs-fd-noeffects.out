ECHO: should have 5000 DP and 9000 DP
FIELD1:Inherited,DW1-Vanillamon,FACEDOWN,CS1-Papershield,CS1-Papershield,FACEDOWN,CS2-Doggiedagger,5000DP Red,CS2-Shieldsmasher,9000DP
FIELD2:
GAME:P1 T3 HATCHING 5
ECHO: -- main, gain just 1 mem
FIELD1:Inherited,DW1-Vanillamon,FACEDOWN,CS1-Papershield,CS1-Papershield,FACEDOWN,CS2-Doggiedagger,5000DP Red,CS2-Shieldsmasher,9000DP
FIELD2:
GAME:P1 T3 MAIN 6
